src/FlipMove.js

```diff
@@ -574,15 +587,28 @@ class FlipMove extends Component<ConvertedProps, FlipMoveState> {
       // If the child element returns null, we need to avoid trying to
       // account for it
       if (!childData.domNode || !child) {
+        childrenBoundingBoxes.push();
         return;
       }
 
-      this.setChildData(childKey, {
-        boundingBox: getRelativeBoundingBox({
+      childrenBoundingBoxes.push(
+        getRelativeBoundingBox({
           childDomNode: childData.domNode,
           parentDomNode,
           getPosition: this.props.getPosition,
         }),
+      );
+    });
+
+    this.state.children.forEach((child, index) => {
+      const childKey = getKey(child);
+
+      if (!childKey || !childrenBoundingBoxes[index]) {
+        return;
+      }
```

Comment:
Nit: This is a super-nit, but I wonder if it makes sense to give that second new condition a variable name, to signify what it represents. Something like:

```js
const wasAbleToPopulateBoundingBox = !!childrenBoundingBoxes[index];

if (!childKey || !wasAbleToPopulateBoundingBox) {
  return;
}
```